plugin.rb

```diff
@@ -224,17 +224,18 @@ def accepted_answer
         {
           post_number: info[0],
           username: info[1],
+          excerpt: info[2]
         }
       end
     end
 
     def accepted_answer_post_info
       # TODO: we may already have it in the stream ... so bypass query here
-
-      Post.where(id: accepted_answer_post_id, topic_id: object.topic.id)
-          .joins(:user)
-          .pluck('post_number, username')
-          .first
+      post = Post.where(id: accepted_answer_post_id, topic_id: object.topic.id).joins(:user).first
+      excerpt = post.excerpt
```

Comment:
you can pluck, check out the implementation, just call PrettyText direct 

https://github.com/discourse/discourse/blob/master/app/models/post.rb#L366